src/snowflake/snowpark/modin/plugin/PANDAS_CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - Fixed UDTF "int64 is not serializable" errors from new Snowflake release in `apply(axis=1)`
 - Fixed binary operations with single-row DataFrame and Series.
 - Fixed incorrect return type in `qcut` when given `Series` input and improved error checking logic.
+- Fixed support for `display.max_rows` and `display.max_cols` being set to `None`.
```

Comment:
can you rebase? This file is OFD.

Reply:
can you move under 1.16.0a1? 1.15.0a1 has been released.

Reply:
done!